lib/publiccloud/basetest.pm

```diff
@@ -110,15 +110,19 @@ sub _cleanup {
     my ($self) = @_;
     die("Cleanup called twice!") if ($self->{cleanup_called});
     $self->{cleanup_called} = 1;
+    record_info 'XXX';
```

Comment:
looks like some debug code which you forgot to clean up

Reply:
and two other record_info below looks the same

Reply:
Yeah, I marked the same but then realized that it's marked as notready therefore Pavel is probably still fiddling around here :slightly_smiling_face: